sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -217,4 +214,52 @@ def get_queue_receiver(self, queue_name, **kwargs):
             **kwargs
         )
 
-        return receiver
+    def get_queue_session_receiver(self, queue_name, session_id=None, **kwargs):
```

Comment:
need to add get_subscription_session_receiver as well

Reply:
Yep, see other comment about "our PRs crossed in flight", the sub changes should be dealt with.